### ISSUE TITLE
Harmonizing Ancient Greek Treebanks

### ISSUE THREAD

**Issue by mr-martian** [unipv-larl/UD4HL#7]
I'm starting a thread here for getting the 3 Ancient Greek treebanks up to date and in sync.

At present we have, I think:
- PROIEL
  - Converted from the PROIEL scheme, as far as I know the most consistent with the UD guidelines
- Perseus
  - Converted from AGLDT, progress on revision presented at UD4HL on June 30
  - Conversion from UDv1 to UDv2 possibly incomplete
- PTNK
  - Initial annotations about to begin, via word-aligning the Septuagint with the Ancient Hebrew treebank and manually correcting the results
  - The current contents of the repo were partially converted from an existing treebank in Perseus format which only covers Genesis, hence the change in methodology

**Comment by francescomambrini** [comment]
I think it is important that we agree on a series of principles, so that we can work together to a unified set of guidelines (which, by the way, we should prepare as part of the UD ufficial distribution).

For instance, the [guidelines](https://universaldependencies.org/u/pos/DET.html) of the `DET` POS recommend that all languages specify the lexical items that can be labeled as determiners (as that of DET should be a closed class). I think we should use this space to discuss such questions and reach an agreement.

If you're OK with that approach, I am going to open a series of issues related to topic that should be covered in the Ancient Greek guidelines (or, in any case, be clarified for any future UD annotation of Ancient Greek), and make reference to this general discussion. In this way, we could keep track of all that is being discussed. I'll also try to be specific, and list  
the situation in the UD v. 2.12.

**Comment by mr-martian** [comment]
Agreed. Discussing everything in one thread would get unwieldy very quickly.

**Comment by mr-martian** [comment]
In light of https://github.com/UniversalDependencies/docs/issues/954#issuecomment-1623979419, do we perhaps want to make an Ancient Greek Harmonization project on the UD docs repo?

**Comment by mr-martian** [comment]
I have started writing a udapi module for language-specific validation. If anyone has ideas for more checks or wants write access, let me know: https://github.com/mr-martian/UD-GreekCheck

**Comment by mr-martian** [comment]
Project created: https://github.com/orgs/UniversalDependencies/projects/6
